src/bentoml/_internal/types.py

```diff
@@ -255,6 +255,12 @@ def isinstance(self, obj: t.Any) -> t.TypeGuard[T]:
         except ValueError:
             return False
 
+    def issubclass(self, klass: type) -> bool:
+        try:
+            return issubclass(klass, self.get_class(import_module=False))
+        except ValueError:
+            return False
```

Comment:
Should we also support sth more lenient when calling issubclass with `GenericAlias`?

This won't work with `LazyType(list[t.Any]).issubclass(list)`?

Reply:
I think we can add this later, not very high priority atm